src/amo/withExperiment.js

```diff
@@ -201,10 +206,17 @@ export const withExperiment = ({
 
         if (variantToStore.id !== NOT_IN_EXPERIMENT) {
           // send an enrollment event
-          _tracking.sendEvent({
+          const event = {
             action: variantToStore.id,
             category: [EXPERIMENT_ENROLLMENT_CATEGORY, id].join(' '),
-          });
+          };
+
+          if (_config.get('server')) {
```

Comment:
I guess we cannot really do this in `tracking`?

Reply:
That's an interesting idea which I hadn't tried. Maybe we can? I'll give that a try.

Reply:
I played around with this a bit, and there doesn't seem to be a simple way to give `tracking` access to the Redux store in order to be able to dispatch an action, and read the queue from state. It's doable, but it would require changing the way `tracking` works, and is instantiated from other components, and it's not something I think is worth tackling right now. I'll open a follow-up to look into making that change, but for now I think this is probably the most pragmatic approach.

Reply:
It's funny how it's always after writing a comment like the one above, that I figure out there actually *is* a way to do it fairly easily. 😛 I'll update the patch with this new method, which will require some changes to the way `sendEvent` is called, but I think it will still be worthwhile.